From 408aae556947f10fa87536e01da83b74766f2eba Mon Sep 17 00:00:00 2001
From: Vitaly Margolin <vmargolin@habana.ai>
Date: Sun, 25 Aug 2024 13:55:04 +0300
Subject: [PATCH] [SW-195340] g3: ppboot: increase ppboot size

Increase ppboot size for Gaudi3 from 18K to 20K
in order to implement watchdog timer support.
After this patch the amount of free memory in ppboot will be ~2kb.
According to the estimation, since we don't use secondary images
fallback during boot, the boot time after this change will be ~77ms.

Signed-off-by: Vitaly Margolin <vmargolin@habana.ai>
Change-Id: I9b9864aa41c391079bc60acec929071b6559f86b
---

diff --git a/arch/gaudi3/arc/Makefile b/arch/gaudi3/arc/Makefile
index 8e40add..c1cab70 100644
--- a/arch/gaudi3/arc/Makefile
+++ b/arch/gaudi3/arc/Makefile
@@ -1,6 +1,6 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
-PPBOOT_MAX_SIZE := 18432
+PPBOOT_MAX_SIZE := 20480
 
 PHY_FW_SIZE ?= 0	#dummy value for building auxilary apps w/o phy logic
 
diff --git a/arch/gaudi3/arc/common.h b/arch/gaudi3/arc/common.h
index 645676c..60cac08 100644
--- a/arch/gaudi3/arc/common.h
+++ b/arch/gaudi3/arc/common.h
@@ -8,7 +8,7 @@
 
 #include <sizes.h>
 
-#define PPBOOT_SIZE			0x4800		/* 18KB */
+#define PPBOOT_SIZE			0x5000		/* 20KB */
 #define PREBOOT_SIZE			0x20000		/* 128KB */
 
 #define HL_FW_BOOT_PTRS_BASE		0x60000
